build.sbt

```diff
@@ -18,6 +18,7 @@ libraryDependencies ++= Seq(
     ExclusionRule(organization = "com.rojoma"),
     ExclusionRule(organization = "com.socrata", name = "socrata-thirdparty-utils_2.10")),
   "com.socrata"             %% "socrata-thirdparty-utils" % "3.0.0",
+  "com.socrata"             %% "soql-pack"                % "0.6.0",
```

Comment:
It's still needed for tests.  However, one could make the case that the test code could be pushed into a library too.